metric_fu.gemspec

```diff
@@ -67,5 +67,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency "test_construct"
   # ensure we have a JSON parser
   s.add_development_dependency "json"
-  s.add_development_dependency "simplecov", "~> 0.9"
+  s.add_development_dependency "simplecov", "< 0.18"
```

Comment:
Is this change what fixed the CI?

Reply:
@bronzdoc Yes, `simplecov` v0.18.x is not compatible with the current version of `metric_fu`